bin/processGsAndCopyRawDataToPrm.sh

```diff
@@ -0,0 +1,652 @@
+fi
+
+if [[ -e "${_controlFileBase}.sanityChecking.finished" && "${_controlFileBase}.sequencingStartDate" ]]
+then
+	#
+	# If previous sanityChecking was ok, skip
+	# 
+	log4Bash 'DEBUG' "${LINENO}" "${FUNCNAME:-main}" '0' "${_controlFileBase}.sanityChecking.finished is there, ready for samplename renaming."
+	IFS=$'\n' _sequencingStartDate=($( cat "${_controlFileBase}.sequencingStartDate" | sort | uniq ))
+else
+	log4Bash 'TRACE' "${LINENO}" "${FUNCNAME:-main}" '0' "${_controlFileBase}.sanityChecking.finished not present. return..."
+	return
+fi
+
+echo "_sequencingStartDate: ${_sequencingStartDate}"
+
+cd "${_runPrefix}"
+local _firstFastQ=$(ls -1 *.fastq.gz | head -1)
+	echo "DEBUG:    Found _firstFastQ ............ = ${_firstFastQ}"
+
+	local _regex='^([A-Z0-9]+)_(103373-009)(.+).fastq.gz'
+
+	if [[ "${_firstFastQ}" =~ ${_regex} ]]
+	then
+		local _flowcell="${BASH_REMATCH[1]}"
+		local _runID="${BASH_REMATCH[2]}"
+
+		echo "DEBUG:    Found _flowcell ............... = ${_flowcell}"
+		echo "DEBUG:    Found _runID .................. = ${_runID}"
+	else
+		echo "FATAL: Failed to parse required meta-data values from ID of first read of ${_runPrefix}/${_firstFastQ}"
+		exit 1
+	fi
+
+module load ngs-utils/"${NGS_UTILS_VERSION}"
+module list
+
+cd "${_runPrefix}"
+
+renameFastQs.bash -s "${_sequencingStartDate}" -f "${_flowcell}_${_runID}"'*'
+
+if [ ${?} -eq 0 ]
+then
+	touch "${_controlFileBase}.renameFastQs.finished"
+	log4Bash 'DEBUG' "${LINENO}" "${FUNCNAME:-main}" '0' "renameFastQs finished."
+else
+	touch "${_controlFileBase}.renameFastQs.failed"
+	log4Bash 'FATAL' "${LINENO}" "${FUNCNAME:-main}" '0' "renameFastQs failed."
+fi
+
+}
+
+function mergeSamplesheetPerProject() {
+	
+	local _run="${1}"
+	local _sampleSheet="${TMP_ROOT_DIR}/Samplesheets/archive/${_run}.${SAMPLESHEET_EXT}"
+	local _controlFileBase="${TMP_ROOT_DIR}/logs/${_run}/${_run}.${SCRIPT_NAME}"
+	local _controlFileFinished="${_controlFileBase}.mergeSamplesheetPerProject.finished"
+	local _logFile="${_controlFileBase}.log"
+	local _processSoFarSoGood='false'
+	
+	if [[ -e "${_controlFileFinished}" ]]
+	then
+		log4Bash 'DEBUG' "${LINENO}" "${FUNCNAME:-main}" '0' "Found ${_controlFileFinished} -> Skipping merging ${_run}."
+		return
+	elif [[ ! -e "${_controlFileBase}.sanityChecking.finished" || ! -e "${_controlFileBase}.renameFastQs.finished" ]]
+	then
+		log4Bash 'FATAL' "${LINENO}" "${FUNCNAME:-main}" '0' "Not Found ${_controlFileBase}.sanityChecking.finished or ${_controlFileBase}.renameFastQs.finished"
+		touch "${_controlFileBase}.mergeSamplesheetPerProject.failed"
+	else
+		log4Bash 'DEBUG' "${LINENO}" "${FUNCNAME:-main}" '0' "No ${_controlFileFinished} present -> Merge samplesheets for ${_run}..."
+		_processSoFarSoGood='true'
+		
+	fi
+	
+
+#
+# combine GS samplesheet with inhouse samplesheets.
+python NGS_Automated/bin/createInhouseSamplesheetFromGS.py \
```

Comment:
When createInhouseSamplesheetFromGS.py is loaded with a proper module, has a proper shebang and has the right permissions you can remove:
```python NGS_Automated/bin/```